### PR TITLE
fix(CORE-965): helm/helm repository/annotations group/labels group  data source

### PR DESCRIPTION
**Pull request by pggb25**

### CODE REVIEW

**Review by benjaminch — APPROVED** [review]

File: qovery/data_source_helm.go [hunk 1]
```diff
@@ -274,6 +280,196 @@ func (d helmDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, re
 					},
 				},
 			},
+			"timeout_sec": schema.Int64Attribute{
+				Description: "Helm timeout in second",
```
Comment (benjaminch) [inline qovery/data_source_helm.go:284]: _typo_: seconds

File: qovery/data_source_helm.go [hunk 2]
```diff
@@ -274,6 +280,196 @@ func (d helmDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, re
...
+				Optional:    true,
+				Computed:    true,
+			},
+			"auto_preview": schema.BoolAttribute{
+				Description: "Specify if the environment preview option is activated or not for this helm.",
```
Comment (benjaminch) [inline qovery/data_source_helm.go:289]: _suggestion_: `specify if environment preview option is activated or not for this helm.`

File: qovery/data_source_helm.go [hunk 3]
```diff
@@ -274,6 +280,196 @@ func (d helmDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, re
...
+				Optional:    true,
+				Computed:    true,
+			},
+			"auto_deploy": schema.BoolAttribute{
+				Description: " Specify if the service will be automatically updated on every new commit on the branch.",
```
Comment (benjaminch) [inline qovery/data_source_helm.go:294]: _suggestion_: remove extra starting space + extra `the` => `Specify if service will be automatically updated on every new commit on branch.`

File: qovery/data_source_helm.go [hunk 4]
```diff
@@ -274,6 +280,196 @@ func (d helmDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, re
...
+					},
+					"set_string": schema.MapAttribute{
+						ElementType: types.StringType,
+						Optional:    true,
+					},
+					"set_json": schema.MapAttribute{
+						ElementType: types.StringType,
+						Optional:    true,
+					},
+					"file": schema.SingleNestedAttribute{
```
Comment (benjaminch) [inline qovery/data_source_helm.go:373]: _suggestion_: Remove extra `the` => `Define overrides by selecting a YAML file from a git repository (preferred) or by passing raw YAML files.`

File: qovery/data_source_helm.go (hunk 1 above)
Comment (pggb25) [reply qovery/data_source_helm.go]: done

File: qovery/data_source_helm.go (hunk 3 above)
Comment (pggb25) [reply qovery/data_source_helm.go]: done

File: qovery/data_source_helm.go (hunk 4 above)
Comment (pggb25) [reply qovery/data_source_helm.go:373]: done

File: qovery/data_source_helm.go (hunk 2 above)
Comment (pggb25) [reply qovery/data_source_helm.go:289]: done

**Review by benjaminch — APPROVED** [review]

**Pull request merged**